collectors/ciscoduo/collector.js

```diff
@@ -178,10 +182,11 @@ class CiscoduoCollector extends PawsCollector {
             };
         } else {
             //There is no next page concept for this API, So Setting up the next state mintime using the last log (Unix timestamp + 1).
+            // call after 60 sec to avoid multiple api call
```

Comment:
I think the actual issue is [here](https://github.com/alertlogic/paws-collector/blob/aa2343b20b9e208393ca11de3d1fa20ad7ee4b32/collectors/ciscoduo/utils.js#L42-L46)
Please check if we can avoid using pagination for continuous time interval polling.
Pagination is a way to get all the data which collector queries within one collection interval. And collection intervals should be moved separately from pagination.